oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/OAuth2LoginAuthenticationFilter.java

```diff
@@ -147,7 +148,7 @@ public final void setAuthorizationRequestRepository(AuthorizationRequestReposito
 		this.authorizationRequestRepository = authorizationRequestRepository;
 	}
 
-	private OAuth2AuthorizationResponse convert(HttpServletRequest request) {
```

Comment:
This needs to remain `private`.

Reply:
@jgrandja I couldn't verify `convert()` invocation easily without this change and I thought it's a good compromise to be package-private to make it easy to test. If there's any way not to do this, please let me know.